tests/unit/bokeh/plotting/test___init___plotting.py

```diff
@@ -42,6 +43,7 @@
     'output_notebook',
     'reset_output',
     'row',
+    'Row',
```

Comment:
alternatively, we could remove `Row` and `Column` but that would have been better done at 3.0

Reply:
Let's keep the status quo for now. If anyone feels strongly about removing these from `bokeh.plotting` please make an issue.